django/contrib/auth/tests/custom_user.py

```diff
@@ -221,6 +223,11 @@ class CustomUserBadRequiredFields(AbstractBaseUser):
     class Meta:
         app_label = 'auth'
 
+
+class CustomUserUUIDasPK(AbstractUser):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
```

Comment:
I have fixtures for test, default value just for explicit\right field declaration.